src/Symfony/Component/Intl/Data/Generator/RegionDataGenerator.php

```diff
@@ -125,14 +137,25 @@ protected function generateDataForRoot(BundleEntryReaderInterface $reader, $temp
     protected function generateDataForMeta(BundleEntryReaderInterface $reader, $tempDir)
     {
         $rootBundle = $reader->read($tempDir, 'root');
+        $metadataBundle = $reader->read($tempDir, 'metadata');
 
         $this->regionCodes = array_unique($this->regionCodes);
 
+        $alpha2ToAlpha3 = $this->generateAlpha3($metadataBundle);
+
         sort($this->regionCodes);
 
+        $alpha3ToAlpha2 = [];
```

Comment:
cant we do `$alpha3ToAlpha2 = array_flip($alpha2ToAlpha3)` like in languages?

Reply:
I just wanted to be 100% sure that we had all the entries that was in `$this->regionCodes`.

Since this is a generator I want to code to be accurate instead of efficient.